test/extended/authorization/podsecurity_admission.go

```diff
@@ -189,14 +193,302 @@ var _ = g.Describe("[sig-auth][Feature:PodSecurity][Feature:SCC]", func() {
 	})
 })
 
+var _ = g.Describe("[sig-auth][Feature:PodSecurity][Feature:LabelSyncer][Feature:ForceHistoricalOwnership]", func() {
+	defer g.GinkgoRecover()
+
+	ctx := context.Background()
+	oc := exutil.NewCLIWithoutNamespace("pod-security-managed-labels-historical")
+
+	cpc := "cluster-policy-controller"
+	psaLabelSyncer := "pod-security-admission-label-synchronization-controller"
+	ocManager := "openshift-tests"
+	labelSyncLabel := "security.openshift.io/scc.podSecurityLabelSync"
+	psaLabels := []string{
+		psapi.EnforceLevelLabel,
+		psapi.EnforceVersionLabel,
+		psapi.AuditLevelLabel,
+		psapi.AuditVersionLabel,
+		psapi.WarnLevelLabel,
+		psapi.WarnVersionLabel,
+	}
+
+	g.It("forcing PSa label historical ownership", func() {
+		for _, testCase := range []struct {
+			name                                   string
+			podSecurityLabelSync                   string
+			expectedManagerForPSaLabelsAfterCreate string
+			expectedManagerForPSaLabelsAfterUpdate string
+		}{
+			{"labelsyncer-off", "false", cpc, ocManager},
+			{"labelsyncer-on", "true", psaLabelSyncer, psaLabelSyncer},
+			{"labelsyncer-undef", "", psaLabelSyncer, ocManager},
```

Comment:
Nit / Not a change request:
I was a little surprised that `expectedManagerForPSaLabelsAfterCreate` is not always the test client, but then I saw that we specify `cpc` as FieldManager.

Reply:
Yeah this was done to simulate that CPC is always the one creating the NS initially; I decided to not define a field in the test parameters that would always be the same, for the sake of simplicity of the test definition.